wolox-books/src/app/app.component.ts

```diff
@@ -1,10 +1,32 @@
 import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
+
 export class AppComponent {
-  title = 'app';
+  form: FormGroup;
+  post: any[];
```

Comment:
You can remove from this variable to the one in line 17, just leave the form since it already stores the values